### DISCUSSION TITLE
WinMerge Archive compare

### DISCUSSION THREAD

**Discussion by modrallm** [WinMerge/winmerge#2331] [General]
Hi...

I've got two versions of a program that generate large numbers of zip files (over 100K).  I wanted to compare the outputs of two runs over the same data set to see that they are producing the same results.

Of course my first thought was WinMerge.

Unfortunately, what I'm seeing is that every single zip is showing different at the top level, but when I click into the comparison it's showing the interior contents are identical.

What factors go into Winmerge showing the zip file as different while showing the contents are identical?  Perhaps the different timestamps on the files?  I saw the setting for ignoring date stamps that were close but not quite, but it seems what I'd need is "ignore date stamps entirely".

I'd hoped to use Winmerge to point out any discrepancies,, but having to drill into each archive (because all look different on the top level) isn't practical with the number of files.

**Comment by sdottaka** [comment]
If the Compare method is to compare the modification date and time of files such as "Modified Date and Size", please select "Full Contents" as the comparison method as shown below and try comparing again.

![image](https://github.com/WinMerge/winmerge/assets/98126/215ce28a-d13a-49a8-a74e-af0567dd9532)
